memberships/views.py

```diff
@@ -141,3 +145,24 @@ def stripe_webhook(request):
         return stripe_webhook.handle(event)
     except ValueError as e:
         return HttpResponse("Failed to parse stripe payload", status=400)
+
+
+@login_required()
```

Comment:
If we are not using this view could we remove it.

Reply:
Which view are you referring to? The one that had `pass` in it? It no longer has pass in it :smile: